mods/ra/rules/vehicles.yaml

```diff
@@ -811,14 +811,10 @@ CTNK:
 		RevealGeneratedShroud: False
 	RevealsShroud@GAPGEN:
 		Range: 4c0
-	Armament@PRIMARY:
-		Weapon: APTusk
-		LocalOffset: -160,-276,232
-		LocalYaw: 60
-	Armament@SECONDARY:
+	Armament:
 		Weapon: APTusk
-		LocalOffset: -160,276,232
-		LocalYaw: -60
+		LocalOffset: -160,-276,232, -160,276,232
```

Comment:
you can't use a list syntax? mini yaml doesn't suppor it?

i.e. is this 2 x a vec3 or something else. 

perhaps require the adding of braces? (x, y, z), (x, y, z).

if you accept it there, you may introduce it in many locations in future.

Reply:
this is not introducing anything new, this PR doesn't even touch `LocalOffset`

Reply:
> i.e. is this 2 x a vec3 or something else.
Yes, even two vec3 is divided by `,`, that is a very old feature which is unrelated with this PR.